inc/class-upgrade.php

```diff
@@ -746,9 +747,18 @@ private function upgrade_149() {
 	private function upgrade_151() {
 		$this->set_home_url_for_151();
 		$this->move_indexables_indexation_reason_for_151();
-
-		add_action( 'init', [ $this, 'set_permalink_structure_option_for_151' ] );
 		add_action( 'init', [ $this, 'store_custom_taxonomy_slugs_for_151' ] );
+		add_action( 'init', [ $this, 'set_permalink_structure_options_for_151' ] );
+	}
+
+	/**
+	 * Performs the 15.3 upgrade.
+	 *
+	 * @return void
+	 */
+	private function upgrade_153() {
+		WPSEO_Options::set( 'category_base_url', 'category' );
+		WPSEO_Options::set( 'tag_base_url', 'tag' );
```

Comment:
Now that I think at this: why are we storing the default values `category` and `tag`? What if users already set custom values _and then_ the upgrade routine runs? In this scenario, the stored values wouldn't reflect the actual used values.